.github/workflows/ci.yml

```diff
@@ -12,7 +12,8 @@ concurrency:
   group: ${{ github.workflow }}-${{ github.ref }}
   cancel-in-progress: true
 jobs:
-  sanityCheck:
+  sanity-check:
+    name: SanityCheck
```

Comment:
I don't think the name matters. but *changes* matter. Every time you submit a change to a project, someone has to review it. We have a branch rule on this job, I expected this to break it. I checked, and I was wrong. It's keyed on `final-status`, not `sanity-check`, so this change is fine. But the name doesn't matter, and changing it took a couple minutes of my time.

I am asking you to submit PRs which provide **end user value**. Fix a bug, add a feature. No more internal stirring, it's consuming too much of my time. I ***STILL*** haven't been able to look at the actual features people are submitting because I'm trying to close these loops on internal QC with you.